### DISCUSSION TITLE
Any good resources to learn GitHub Actions

### DISCUSSION THREAD

**Discussion by arpangreat** [github/india#33] [Satellite India 2021 Q&A] [answered]
Hey fellow folks :smiling_face_with_three_hearts: , can you provide any good resources to learn GitHub Actions ?

Advanced Thanks :heart:

**Comment by chrispat** [comment, marked as answer]
The GitHub documentation is a good place to start.  However, there are a substantial number of other resources like blogs and YouTube videos.  I have a search setup in twitter for "github actions" and see new resources every day.

**Reply by dhruvg20** [reply]
Shared few resources in the [Welcome to the DevOps Discussion Track](https://github.com/github/india/discussions/6) 🙇🏽

**Comment by arpangreat** [comment]
Thanks Sir :heart:

**Comment by Rishit-dagli** [comment]
Hey @arpangreat ,
I would recommend you take a look at [GitHub Actions Learning Labs](https://lab.github.com/search?q=Actions) and I would recommend you to start with [this lab](https://lab.github.com/githubtraining/github-actions:-hello-world) (PS: I started learning with this lab myself too).

**Reply by arpangreat** [reply]
Thanks :heart:

**Reply by shivamkrpandey** [reply]
@Rishit-dagli  Thanks for sharing.

**Comment by FRTMentor** [comment]
https://www.youtube.com/watch?v=GuuNbzmve8Q
https://www.youtube.com/watch?v=A8U3FXUcR5k
https://www.youtube.com/watch?v=V4WtypyR9vE
Please reach out our support team.
support@futurereadytalent.in
